src/Plugin.php

```diff
@@ -27,7 +28,8 @@
  */
 class Plugin extends AbstractPlugin implements LoopAwareInterface
 {
-    const URL_HANDLER_INTERFACE = 'Phergie\Irc\Plugin\React\Url\UrlHandlerInterface';
+    const URL_HANDLER_INTERFACE  = 'Phergie\Irc\Plugin\React\Url\UrlHandlerInterface';
+    const EVENT_FILTER_INTERFACE = 'Phergie\Irc\Plugin\React\EventFilter\FilterInterface';
```

Comment:
Any particular reason for using a constant versus just referencing `FilterInterface::class` where needed?